translations/Marathi/special-terms.md

```diff
@@ -0,0 +1,42 @@
+﻿Terms
+
+Message : संदेश , Most appropriate synonym for general usage in zulip, <username> ला एक संदेश पाठवा
+Private Message (PM) : खाजगी संदेश , Best translation of private to imply it's meaning , <username> ला खासगी एक संदेश पाठवा
+Group PM : ग्रुप खाजगी संदेश  , Transliteration used for group , <group name> मध्ये एक ग्रुप खाजगी संदेश पाठवा
+Realm : संघटन , used translation of organization for same reason as of chinese translation , एक संघटन बनवा
```

Comment:
realm literally translates to 'क्षेत्र', not 'संघटन'

Reply:
The choice to use translation of organization instead is also used in Chinese translation, and 'क्षेत्र' is more commonly understood as an area

Reply:
@hrishi3108: I would suggest using 'क्षेत्र' here, which is more appropriate, I think! :)